### DISCUSSION TITLE
how much memory needed

### DISCUSSION THREAD

**Discussion by fendoukobe** [milvus-io/milvus#6405] [Q&A and General discussion]
hi,
I want to know how much memory it takes to load 15 million vector 1024 dimensions into memory?

**Comment by xiaofan-luan** [comment]
https://zilliz.com/sizing-tool sizing tool is gonna to help you on this.

**Reply by fendoukobe** [reply]
thanks,
but,There are no HNSW and FAISS options

**Reply by xiaofan-luan** [reply]
most of the algorithm supported is listed for FAISS. Sorry we don't have support for HNSW, need a enhancement on that

**Reply by fendoukobe** [reply]
thanks for your apply

**Comment by xiaofan-luan** [comment]
@shengjun1985 @shanghaikid  maybe we want to update our sizing tool?
